### PR TITLE
Allow method on plain client

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

File: src/main/java/com/nextcloud/common/OkHttpMethodBase.kt [hunk 1]
```diff
@@ -144,11 +147,31 @@ abstract class OkHttpMethodBase(
...
+        requestHeaders.forEach { (name, value) -> temp.header(name, value) }
+
+        applyType(temp)
+
+        val request = temp.build()
+
+        try {
+            response = client.newCall(request).execute()
+        } catch (ex: IOException) {
+            System.out.println(ex.message)
```
Comment (AndyScherzinger) [inline src/main/java/com/nextcloud/common/OkHttpMethodBase.kt:169]: should be replaced by proper log statement

**Review by AndyScherzinger — DISMISSED** [review]

**Comment by tobiasKaminsky** [comment]
> approved these changes

There is no point in approving/review a "1. developing", as this only gives false positive (e.g. you might not re-check after I switch it to 2. to review and then your valuable feedback is missing).

In this case added sys out log on purpose, to get the real error message:
> java.lang.RuntimeException: javax.net.ssl.SSLHandshakeException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
at com.nextcloud.common.OkHttpMethodBase.execute(OkHttpMethodBase.kt:170)

I will fix it tomorrow.

File: src/androidTest/java/com/owncloud/android/lib/resources/users/StatusIT.kt [hunk 2]
```diff
@@ -114,7 +119,7 @@ class StatusIT : AbstractIT() {
         val statusesList: ArrayList<PredefinedStatus> =
             result.singleData as ArrayList<PredefinedStatus>
         val newCustomStatusMessage = statusesList[2]
-        val clearAt = System.currentTimeMillis() / 1000 + 3600 // in one hour
+        val clearAt = System.currentTimeMillis() / SECOND_IN_MILLIS + ONE_HOUR_IN_MINUTES
```
Comment (AndyScherzinger) [inline src/androidTest/java/com/owncloud/android/lib/resources/users/StatusIT.kt:122]: either drop the ONE_ or also prefix the SECOND_IN_MILLIS :)

**Review by AndyScherzinger — APPROVED** [review]
LGTM 👍

**Pull request merged**